.travis/grep_simple_mistakes.sh

```diff
@@ -77,6 +77,39 @@ for file in $S2N_FILES_ASSERT_NOTNULL_CHECK; do
   done < <(grep -rnE -A 1 "=\ss2n_stuffer_raw_read\(.*\)" $file)
 done
 
+# Assert that we do not set cipher preferences using the "test_all" or
+# "test_all_tls12" string literals. Tests should use the `TEST_ALL`
+# and `TEST_ALL_TLS12` strings that are initialized in s2n_test.h as
+# part of BEGIN_TEST().
```

Comment:
If you merge before https://github.com/awslabs/s2n/pull/2086, you might want to reach out to Ryan to make sure he doesn't break this check.

Reply:
Given the [shift in how we're defining the behavior](https://github.com/awslabs/s2n/pull/2426#discussion_r533776527), I don't think this should be a conflict anymore.